src/cpp/shared/game/game.h

```diff
@@ -221,6 +240,30 @@ extern int gbInCampaign;
 
 extern int gbGameOver;
 
+extern __int16 giUltimateArtifactBaseX;
+extern __int16 giUltimateArtifactBaseY;
+extern __int16 giUltimateArtifactRadius;
+
+extern char gpcBeforeGSpellLimits[];
+extern int giPlayerInitialResourcesHuman[NUM_DIFFICULTIES][NUM_RESOURCES];
+extern int giPlayerInitialResourcesAI[NUM_DIFFICULTIES][NUM_RESOURCES];
+
+extern int __fastcall FindLastToken(const char *a1, char a2);
+extern char __fastcall StrEqNoCase(int *a1, int *a2);
+
+extern int gbInNewGameSetup;
+extern int iLastMsgNumHumanPlayers;
+extern bool gbSetupGamePosToRealGamePos[];
+extern char gcColorToSetupPos[];
+extern char giVisRange[];
+extern char xIsPlayingExpansionCampaign;
+extern int giCurTurn;
+
+extern __int16 creatureTypesAndQuantities[NUM_FACTIONS][3][3];
+extern int neutralTownCreatureTypes[NUM_FACTIONS][5];
```

Comment:
This needs a more descriptive name. This has something to do with randomly generating the creatures guarding them, right?

Reply:
Yes. I'm not sure what information I should give in the name that isn't there.

Reply:
"Creature types and quantities?" So...every hero's army contains creature types and quantities. So would a list of all the creeps on the map. This is kinda like naming a variable holding the balance of my bank and investment accounts "currency types and amounts."

Reply:
How about "randHeroArmyBounds"?

Reply:
Oh, these comments aren't quite in line with the the things you were trying to highlight. I thought you were talking about neutralTownCreatureTypes.